src/Annotation/Mutation.php

```diff
@@ -17,5 +17,5 @@ final class Mutation extends Field
      *
      * @var array<string>
      */
-    public $targetType;
+    public array $targetType;
```

Comment:
we should keep the typehint `string[]` it give an useful information.

Reply:
`array<string>` and `string[]` are equal.

Reply:
The first diff was removing it not replacing it. But yes I know both are the same.

Reply:
I didn't remove it. I never remove annotations, that provide extra information.